openmetadata-ui/src/main/resources/ui/src/utils/StringsUtils.ts

```diff
@@ -188,7 +188,9 @@ export const customServiceComparator = (a: string, b: string): number => {
 export const replacePlus = (fqn: string) => fqn.replaceAll('+', ' ');
 
 export const ES_RESERVED_CHARACTERS: Record<string, string> = {
-  '+': '\\+',
```

Comment:
Hi this is breaking at other places.
The idea behind this method is just to escape special characters.
We should not be encoding characters here.

Reply:
Refer Glossary Assets page.
Glossary terms with special names are not able to fetch because we are encoding here.

Reply:
Hi!
The error may occur because when querying the api /api/v1/search/query, the searchValue must be converted using the escapeESReservedCharacters method.
If you are wrong, describe the problem in more detail